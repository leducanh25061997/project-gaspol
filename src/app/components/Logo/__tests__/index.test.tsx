import * as React from 'react';
import { render } from '@testing-library/react';

import { Logo } from '..';

describe('<Logo  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Logo />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
