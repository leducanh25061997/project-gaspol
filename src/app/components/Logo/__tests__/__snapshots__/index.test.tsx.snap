// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo  /> should match snapshot 1`] = `
<div
  class="MuiBox-root css-1janyvp"
>
  <p
    class="MuiTypography-root MuiTypography-body1 css-ksnwlb-MuiTypography-root"
  >
    GASPOL!
  </p>
  <span
    class="MuiTypography-root MuiTypography-body1 css-8mnqxp-MuiTypography-root"
  >
    By IMI
  </span>
</div>
`;
